account_check_printing_aeroo/print_stub_lines.py

```diff
@@ -37,14 +37,14 @@ def _aeroo_check_make_stub_line(self, invoice):
             invoice_payment_reconcile = (
```

Comment:
Salut,
Je comprends pas ce que fait cet itérateur? Est ce que tu pourrais m'expliquer ou exploser le code pour le rendre plus accessible?
En plus, je comprends pas pourquoi ça règle le bug.

Merci